src/main/java/ru/lanwen/diff/uri/UriDiffer.java

```diff
@@ -28,22 +28,25 @@
  * User: lanwen
  */
 public class UriDiffer {
+
     //TODO filter
     //TODO encoding
 
-    private URI actual = URI.create("/");
-    private URI expected = URI.create("/");
+    public static final URI DEFAULT_URI = URI.create("/");
```

Comment:
Actually, I've planned to update PR and add unit test for this default value. Should I change access modifier or add unit test?